### PR TITLE
net-libs/libwebsockets: version bump to 3.2.0

### PR DESCRIPTION
**Pull request by jkroonza**
Aimed primarily at #696554.

However, I'm unable to reproduce any of the other reported issues after
this bump.

Closes: https://bugs.gentoo.org/696554
Closes: https://bugs.gentoo.org/615532
Closes: https://bugs.gentoo.org/618428
Closes: https://bugs.gentoo.org/672194
Closes: https://bugs.gentoo.org/669152
Package-Manager: Portage-2.3.76, Repoman-2.3.16
Signed-off-by: Jaco Kroon <jaco@uls.co.za>

### CODE REVIEW

File: net-libs/libwebsockets/libwebsockets-3.2.0.ebuild [hunk 1]
```diff
@@ -0,0 +1,79 @@
...
+
+LICENSE="LGPL-2.1"
+SLOT="0/${PV}"
+KEYWORDS="~amd64 ~arm ~x86"
+IUSE="access-log caps cgi client dbus generic-sessions http-proxy http2 ipv6
+	lejp libev libevent libressl libuv peer-limits server-status smtp socks5
+	sqlite3 ssl static-libs threads zip"
+
+REQUIRED_USE="
+	http-proxy? ( client )
```
Comment (juippis) [inline net-libs/libwebsockets/libwebsockets-3.2.0.ebuild]: There's some alphabetical ordering in here and RDEPEND list, but rest looks good.

File: net-libs/libwebsockets/libwebsockets-3.2.0.ebuild [hunk 2]
```diff
@@ -0,0 +1,79 @@
...
+	generic-sessions? ( sqlite3 )
+	generic-sessions? ( smtp )
+	smtp? ( libuv )
+	?? ( libev libevent )
+"
+
+RDEPEND="
+	sys-libs/zlib
+	http-proxy? ( net-libs/libhubbub )
+	caps?       ( sys-libs/libcap )
```
Comment (juippis) [inline net-libs/libwebsockets/libwebsockets-3.2.0.ebuild]: You could also drop the stylistic style, and follow skel style instead. Check subslots for these if they get linked during compile.

Comment (jkroonza) [reply net-libs/libwebsockets/libwebsockets-3.2.0.ebuild]: I believe I've got this now.  If depended on package does anything other than SLOT="0" then I simply set this to := ... so I hope that's ok.

Comment (juippis) [reply net-libs/libwebsockets/libwebsockets-3.2.0.ebuild]: Not quite... you should check how the binaries and libraries are linked... but generally I do it like that if I'm unsure.

**Pull request closed without merging**